docs/content/en/docs/reference/changelog.md

```diff
@@ -3,12 +3,13 @@ title: "What's New?"
 linkTitle: "What's New?"
 weight: 35
 ---
+## [v0.11.0](https://github.com/aws/eks-anywhere/releases/tag/v0.11.0)
 
-## Unreleased
```

Comment:
We still need this Unreleased section at the top, whether it's in this PR or a follow up one. We don't want this section in the release branch though

Reply:
Ok good catch

Reply:
fixed